test/cut/test_cut_trim_to_supervisions.py

```diff
@@ -363,15 +363,15 @@ def test_cut_trim_to_supervisions_keep_overlapping_extend(mono_cut):
     # Extended on the right side only by (4.0 - 3.37) / 2 == 0.315;
     # the left side is capped by the start of the recording.
     assert len(c1.supervisions) == 1
-    assert c1.start == 0.0
+    assert c1.start == 1.0
```

Comment:
After changing the `mono_cut` fixture, this test case no longer represents the illustrated scenario. Also the comment above seems to be in conflict with the test outcome (looks like `c1` should have 4 seconds now that it's not capped by the start of the recording anymore)

Reply:
Good point. I'll fix the tests tomorrow.

Reply:
Sorry I lost track of this. Have made the appropriate changes now --- should not affect the older tests.